docs/assets/scss/schema-docs.scss

```
@@ -330,4 +330,34 @@
       @include u-text('bold');
     }
   }
+}
+
+
+.no-anchor-xslt {
+  // : :after
+  font-style: normal;
+  font-variant: normal;
+  font-kerning: auto;
+  font-optical-sizing: auto;
+  font-feature-settings: normal;
+  font-variation-settings: normal;
+  font-weight: normal;
+  font-stretch: normal;
+  font-size: 1em;
+  line-height: inherit;
+  font-family: anchorjs-icons;
+  position: absolute;
+  margin-left: -1em;
+  padding-right: 0.5em;
+}
```

Blocking: can we make the anchor class a name more intuitive than `no-anchor-xslt` as that explains implementations but not the purpose (so only useful and relevant for a little bit, especially given the no-anchor). Also we agreed the comments and `font-family` declaration isn't effective or necessary, so can we just remove it?

Are the other class attributes still necessary for styling or just mimicking the AnchorJS implementation we will be removing as a consequence of this change?

```suggestion
.reference-element-anchor {
  font-style: normal;
  font-variant: normal;
  font-kerning: auto;
  font-optical-sizing: auto;
  font-feature-settings: normal;
  font-variation-settings: normal;
  font-weight: normal;
  font-stretch: normal;
  font-size: 1em;
  line-height: inherit;
  position: absolute;
  margin-left: -1em;
  padding-right: 0.5em;
}
```